I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang-16.txt

```diff
@@ -0,0 +1,8232 @@
+      |  +----|   |  |   /   |   |       ___     \   /---o    |      |
+    --o->| Tx o---o->o--o  5 |    \     /   |   / \ /    |___/       |
+      |  +----|   |  |   \   |     \   |    o--/   \           DEG 2 |
+    <-o--| Rx o   |  |    ---o--\   \--o    |     / \     ___        |
+      |  +----|   |  +-------+   \     |    |    /   \   |   \  Line |
+      |       |   |  LTP 30       \----o  6 |   /     \--o    | LTP 3|
+      |   TTP3|   |                    |    o--/         | 3  o------o->
+      |  +----|   |          /-------->o    |            o    |      |
+    --o->| Tx o---o---------/          |    |            |___/       |
+      |  +----|   |  LTP 40             \___|                  DEG 3 |
+    <-o--| Rx o   |                AD 2                              |
+      |  +----|   |                                                  |
+      |       |   |                                                  |
+      +-------+   |                                                  |
+                  |                                                  |
+                  +--------------------------------------------------+
+
+
+
+
+Beller, et al.           Expires 9 January 2025               [Page 111]
+
+Internet-Draft    Opt. Impairment-Aware Topo YANG Model        July 2024
+
+
+     Figure 30: JSON Example for Restricted Connectivity between ROADM
+                   Add-Drop Ports and ROADM Degree Ports
+
+   {
+     "roadm-path-impairments": [
+       {
+         "roadm-path-impairments-id": 1,
+         "roadm-add-path":
+         "Add path impairments from LTP 10 to LTP 1 or LTP 2 or LTP 3."
+       },
+       {
+         "roadm-path-impairments-id": 2,
+         "roadm-add-path":
+         "Add path impairments from LTP 20 or LTP 30 to LTP 1 or LTP 3,
+          through AD1."
+       },
+       {
+         "roadm-path-impairments-id": 3,
+         "roadm-add-path":
+         "Add path impairments from LTP 20 or LTP 30 to LTP 1 or LTP 2,
+         through AD2."
+       }
+     ],
+
+     "connectivity-matrix"[
+       {
+         "id": 1,
+         "from": {
+           "tp-ref" : 20
+         },
+         "to" :  {
+           "tp-ref" : 1,
+           "additional-ltp": [
+              {
+                 "link-tp-ref": 1,
+                 "roadm-path-impairments": 3
+              },
+              {
+                 "link-tp-ref": 2,
+                 "roadm-path-impairments": 3
+              }
+           ]
+         },
+         "is-allowed": true,
+         "roadm-path-impairments": 2
+       },
+       {
+         "id": 2,
+
+
+
+Beller, et al.           Expires 9 January 2025               [Page 112]
+
+Internet-Draft    Opt. Impairment-Aware Topo YANG Model        July 2024
+
+
+         "from": {
+           "tp-ref" : 20
+         },
+         "to" :  {
+           "tp-ref" : 3,
+           "additional-ltp": [
+              {
+                 "link-tp-ref": 1,
```

Comment:
same comment link-tp-ref should be ltp-ref

Reply:
@EstherLerouzic : correct, we will fix